Installments/index.jsx

```diff
@@ -13,6 +13,59 @@ const classes = {
   cellHighlight: `${baseClass}__cell__highlight`
 }
 
+const TARGET_FPS = 60
+
+const requestAnimationFrame = (fn) => {
+  if (window.requestAnimationFrame) {
+    window.requestAnimationFrame(fn)
+  } else {
+    setTimeout(fn, 1000 / TARGET_FPS)
+  }
+}
+
+const debounce = (fn) => {
+  let waiting = false
+
+  return function (...args) {
+    if (!waiting) {
+      waiting = true
+
+      requestAnimationFrame(() => {
+        fn(...args)
+        waiting = false
+      })
```

Comment:
We already have react-motion in the build (for the smoooth menus), so why all this effort?